indico/web/client/js/utils/redux.js

```diff
@@ -23,31 +24,24 @@ import {composeWithDevTools} from 'redux-devtools-extension';
 // similar to the standard combineReducers, but accepts
 // root reducers as well
 function _combineReducers(...reducers) {
-    return (state, action) => {
-        for (const reducer of reducers) {
-            if (typeof reducer === 'function') {
-                // it's a reducer
-                state = reducer(state, action);
-            } else {
-                state = combineReducers(reducer)(state, action);
-            }
-        }
-        return state;
-    };
-}
+    const reducerFunctions = reducers.map(reducer => (
+        (typeof reducer === 'function') ? reducer : combineReducers(reducer))
+    );
 
+    return (state, action) => reducerFunctions.reduce((st, fn) =>  fn(st, action), state);
```

Comment:
double space after the second `=>`

Reply:
weird... ESLint doesn't seem to care about that.